src/util/unicode.h

```diff
@@ -20,7 +20,6 @@ std::string narrow(const std::wstring &s);
 std::wstring widen(const std::string &s);
 
 std::string utf32_to_utf8(const std::basic_string<unsigned int> &s);
```

Comment:
Yes, the renamed one was `utf32_to_utf8(unsigned int c, std::string &result)`, this one stayed the same. I have made that one `static`.